test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -66,9 +66,9 @@ private void IntermediateData(string dataPath)
             var transformedData = dataPipeline.Fit(data).Transform(data);
 
             // 'transformedData' is a 'promise' of data. Let's actually read it.
-            var someRows = transformedData.AsDynamic
+            var someRows = mlContext
                 // Convert to an enumerable of user-defined type. 
-                .AsEnumerable<InspectedRow>(mlContext, reuseRowObject: false)
+                .CreateEnumerable<InspectedRow>(transformedData.AsDynamic, reuseRowObject: false)
```

Comment:
>CreateEnumerable [](start = 17, length = 16)

You need to update [https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md) with changes in this filea and other CookBookSamples file.

Reply:
Only two examples touched and they are fixed. Please search for `InspectedRow` for static one and `InspectedRowWithAllFeatures` dynamic one.